tests/alerts/test_session_opened_sensitive_user.py

```diff
@@ -126,16 +126,6 @@ class TestSessionOpenedUser(AlertTestSuite):
         )
     )
```

Comment:
I wasn't sure where to put it, but looks like we have a couple alerts that still use the _type field:

$ grep -r '_type' ./alerts/ (with some filtering)
./alerts//amo_failed_logins.py:            TermMatch('_type', 'addons'),
./alerts//cloudtrail_logging_disabled.py:            TermMatch('_type', 'cloudtrail'),
./alerts//ssh_password_auth_violation.py:            TermMatch('_type', 'event'),
./alerts//ssh_ioc.py:            TermMatch('_type', 'event'),
./alerts//host_scanner_alerts.py:            TermMatch('_type', 'cef'), 

I'm also not sure what to do about these entries, thoughts?
./alerts//lib/alerttask.py:                'documenttype': e['_type'],
./alerts//lib/alerttask.py:                    'type': alertResultES['_type'],
./alerts//lib/alerttask.py:                self.es.save_event(index=event['_index'], doc_type=event['_type'], body=event['_source'], doc_id=event['_id'])

Reply:
Once we split the indexes we can remove all remaining calls to _type - this PR is only for removing the _type from the search parameters that trigger events. Low hanging fruit so to speak.

Reply:
These have notes on them in the spreadsheet as to why they aren't modified, many of them don't trigger as they are.

Reply:
as for alerttask.py -  should be removed when we split the indexes.